pyproject.toml

```diff
@@ -18,6 +18,7 @@ python_version = "3.10"
 strict = true
 disallow_untyped_calls = false
 warn_return_any = false
+disallow_subclassing_any = false
```

Comment:
is it because of `abc.ABC`?

Reply:
It's because of subclassing Baker class for our CustomBaker